src/index.js

```diff
@@ -335,7 +348,7 @@ class IPLDResolver {
         if (treePaths.length === 0 && queue.length > 0) {
           ({ cid, basePath } = queue.shift())
           const format = await this._getFormat(cid.codec)
-          block = await this.bs.get(cid)
+          block = await this.bs.get(cid, options)
```

Comment:
Would it make sense to forward only the `signal` value and not all of the `options` object? This is really just a question.

Reply:
It's a fair point, but the way we wrap modules inside modules, if we let the wrapping module be the gatekeeper of what gets passed on, it becomes tied to the API of that module more strongly.

This may cause problems down the line as we're potentially passing more arguments to the wrapped module than strictly necessary, but we trade off against flexibility in that case.